canonical-websites/insights.ubuntu.com
# Comprehensive API functions rewrite

- Rewrite `/archives` route, to use query parameters rather than multiple routes
- Rewrite api.py functions to make it a fairly transparent wordpress API library
- Provide many more helpers in `helpers.py`
- Rewrite `pagination.html`
- Make the use of API functions much more consistent for all routes

QA
--

Run `./run test`, and `./run`. Thoroughly check the site looks and works as it should.
